docs/sources/get-started/quick-start.md

```diff
@@ -194,165 +192,337 @@ You can view your logs using the command line interface, [LogCLI](/docs/loki/<LO
-   1. Calculate the number of logs per second where the JSON field `status` has the value `404`:
+   2. Find all the log lines in the `{container="greenhouse-main_app-1"}` stream that contain the string `POST`:
       <!-- INTERACTIVE copy START -->
-      ```bash
-      sum by(container) (rate({container="evaluate-loki-flog-1"} | json | status=`404` [$__auto]))
+      ```logql
+      {container="greenhouse-main_app-1"} |= "POST"
       ```
       <!-- INTERACTIVE copy END -->
-   The final query is a metric query which returns a time series.
-   This makes Grafana draw a graph of the results.
-
-   You can change the type of graph for a different view of the data.
-   Click **Bars** to view a bar graph of the data.
 
-1. Click the **Builder** tab (3) to return to builder mode in the query editor.
-   1. In builder mode, click **Kick start your query** (5).
-   1. Expand the **Log query starters** section.
-   1. Select the first choice, **Parse log lines with logfmt parser**, by clicking **Use this query**.
-   1. On the Explore tab, click **Label browser**, in the dialog select a container and click **Show logs**.
+<!-- INTERACTIVE page step4.md END -->
 
-For a thorough introduction to LogQL, refer to the [LogQL reference](https://grafana.com/docs/loki/<LOKI_VERSION>/query/).
+<!-- INTERACTIVE page step5.md START -->
 
-## Sample queries (code view)
+### Extracting attributes from logs
 
-Here are some more sample queries that you can run using the Flog sample data.
-
-To see all the log lines that flog has generated, enter the LogQL query:
+Loki by design does not force log lines into a specific schema format. Whether you are using JSON, key-value pairs, plain text, Logfmt, or any other format, Loki ingests these logs lines as a stream of characters. The sample application we are using stores logs in [Logfmt](https://brandur.org/logfmt) format:
 <!-- INTERACTIVE copy START -->
 ```bash
-{container="evaluate-loki-flog-1"}
+ts=2025-02-21 16:09:42,176 level=INFO line=97 msg="192.168.65.1 - - [21/Feb/2025 16:09:42] "GET /static/style.css HTTP/1.1" 304 -"
+```
+To break this down:
+- `ts=2025-02-21 16:09:42,176` is the timestamp of the log line.
+- `level=INFO` is the log level.
+- `line=97` is the line number in the code.
+- `msg="192.168.65.1 - - [21/Feb/2025 16:09:42] "GET /static/style.css HTTP/1.1" 304 -"` is the log message.
+
+<!-- INTERACTIVE copy END -->
+When querying Loki, you can pipe the result of the label selector through a parser. This extracts attributes from the log line for further processing. For example, lets pipe `{container="greenhouse-main_app-1"}` through the `logfmt` parser to extract the `level` and `line` attributes:
+<!-- INTERACTIVE copy START -->
+```logql
+{container="greenhouse-main_app-1"} | logfmt
 ```
 <!-- INTERACTIVE copy END -->
-The flog app generates log lines for simulated HTTP requests.
+When you now expand a log line in the query result, you will see the extracted attributes.
```

Comment:
Since this is a getting started, let's not assume that users know about log details yet.
```suggestion
When you click the arrow (>) next to a log line in the query result to expand the view, you will see the extracted attributes in the log details.
```